tools/universe/package_manager.py

```diff
@@ -0,0 +1,87 @@
+"""
+A simple package manager for Universe packages.
+"""
+import logging
+import subprocess
+import json
+import os
+import tempfile
+try:
+    import requests
+    _HAS_REQUESTS = True
+except ImportError:
+    _HAS_REQUESTS = False
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PackageManager:
+    """A simple package manager for retrieving universe packages"""
+    def __init__(self, universe_url="https://universe.mesosphere.com/repo",
+                 dcos_version="1.10",
+                 package_version="4"):
+
+        self.universe_url = "https://universe.mesosphere.com/repo"
+        self._headers = {
+            "User-Agent": "dcos/{}".format(dcos_version),
+            "Accept": "application/vnd.dcos.universe.repo+json;"
+                      "charset=utf-8;version=v{}".format(package_version),
+        }
+
+        self.__package_cache = None
+
+    def get_packages(self):
```

Comment:
IIRC, python likes double newlines between functions (here and below). Looks like the previous version didn't have them, but may as well fix it while here...